package.json

```diff
@@ -29,11 +29,10 @@
   "homepage": "https://github.com/CanopyTax/node-jspm-jasmine#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-preset-es2015": "^6.6.0"
-  },
-  "peerDependencies": {
-    "jasmine": "*",
-    "jspm": "*"
+    "babel-preset-es2015": "^6.6.0",
+    "commander": "^2.9.0",
+    "jasmine": "^2.4.1",
+    "jspm": "^0.17.0-beta.12"
```

Comment:
Good catch on the commander dep too. Let's just stick with jasmine and jspm being peerDependencies and move commander into dependencies like you said. The Readme walks people through installing those peer dependencies, so I think we're fine even with the npm@3 behavior.